apps/oxlint/src/snapshots/_-c fixtures__config_ignore_patterns__ignore_extension__eslintrc.json fixtures__config_ignore_patterns__ignore_extension__main.js@oxlint.snap

```diff
@@ -5,7 +5,8 @@ source: apps/oxlint/src/tester.rs
 arguments: -c fixtures/config_ignore_patterns/ignore_extension/eslintrc.json fixtures/config_ignore_patterns/ignore_extension/main.js
 working directory: 
 ----------
-Finished in <variable>ms on 0 files using 1 threads.
+Found 0 warnings and 0 errors.
+Finished in <variable>ms on 1 file with 88 rules using 1 threads.
 ----------
-CLI result: LintNoFilesFound
+CLI result: LintSucceeded
```

Comment:
It feels like this should still say `on 0 files`... since it didn't actually lint anything (`fixtures/config_ignore_patterns/ignore_extension/main.js` should've been in the ignore pattern here > https://github.com/oxc-project/oxc/blob/32f860d238ca97dc8691b5fcec680d99ad9aab42/apps/oxlint/fixtures/config_ignore_patterns/ignore_extension/eslintrc.json#L2-L3)